src/plugins/kibana_usage_collection/server/collectors/usage_counters/integration_tests/rollups.test.ts

```diff
@@ -87,31 +92,12 @@ describe('usage-counters', () => {
     internalRepository = start.savedObjects.createInternalRepository([
       USAGE_COUNTERS_SAVED_OBJECT_TYPE,
     ]);
-  });
 
-  it('deletes documents older that the retention period, from all namespaces', async () => {
     // insert a bunch of usage counters in multiple namespaces
-    const old = [
-      createCounter('domain1', 'a', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS + 1),
-      createCounter('domain1', 'b', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS + 1, 'one'),
-      createCounter('domain1', 'b', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS + 1, 'two'),
-      createCounter('domain2', 'a', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS + 1),
-    ];
-
-    const recent = [
-      createCounter('domain1', 'a', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS - 1),
-      createCounter('domain1', 'b', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS - 1, 'one'),
-      createCounter('domain1', 'b', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS - 1, 'two'),
-      createCounter('domain2', 'a', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS - 1),
-      createCounter('domain2', 'c', USAGE_COUNTERS_KEEP_DOCS_FOR_DAYS - 1), // different counterName (does not have an old counterpart)
-    ];
-
-    getCurrentTimeMock.mockReturnValue('2024-06-24T10:00:00.000Z'); // 6 days old
-    await Promise.all(old.map((counter) => incrementCounter(internalRepository, counter)));
-
-    getCurrentTimeMock.mockReturnValue('2024-06-26T10:00:00.000Z'); // 4 days old
-    await Promise.all(recent.map((counter) => incrementCounter(internalRepository, counter)));
+    await createTestCounters(internalRepository);
+  });
 
+  it('deletes documents older that the retention period, from all namespaces', async () => {
```

Comment:
FMI: how are we testing the `from all namespaces` here?

EDIT: Oh! gotcha! `createTestCounters` creates some counters in other spaces.